doc/plugin_server_nodeattestor_awsiid.md

```diff
@@ -0,0 +1,13 @@
+# Server plugin: NodeAttestor "aws_iid"
+
+*Must be used in conjunction with the agent-side aws_iid plugin*
+
+The `aws_iid` plugin automatically attests instances using the AWS Instance Metadata API and the AWS Instance Identity document. It also allows an operator to use AWS Instance IDs when defining SPIFFE ID attestation policies. 
+
+| Configuration | Description | Default                 |
+| ------------- | ----------- | ----------------------- |
+| trust_domain  | The trust domain that the agent belongs to. |  |
+| access_id     | The AWS access secret key id of IAM user with action policy to allow "ec2:DescribeInstances". An ec2 client to introspect the instance being attested is created. | Value of `AWS_ACCESS_KEY_ID` environment variable |
+| secret        | Specifies the AWS access secret key corresponding to the access_id. | Value of `AWS_SECRET_ACCESS_KEY` environment variable |
+| session_id    | Specifies the AWS Session Token Id.  |  |
```

Comment:
What is this used for?

Reply:
For posterity: the session_id is required when temporary access credentials are in use. While it is exposed as a configurable, it's not very useful since the value is only read once, when the plugin starts. Until we have config reload or some way to manage temporary credentials without restarting the server, it makes sense to exclude this configurable from the documentation.